### PR TITLE
feat(atInternet): update tracking

### PR DESCRIPTION
**Pull request by JeremyDec**
## Title of the Pull Requests 
Update tracking in manager telecom

(Linked to these PR : 
- https://github.com/ovh-ux/ovh-angular-actions-menu/pull/8
- https://github.com/ovh-ux/ovh-angular-sidebar-menu/pull/20)

### CODE REVIEW

File: client/app/telecom/dashboard/guides/telecom-dashboard-guides.controller.js [hunk 1]
```diff
@@ -13,6 +13,33 @@ angular.module("managerApp").controller("TelecomDashboardGuidesCtrl", function (
...
+            hit.name = "Setting_Modem";
+            break;
+        case URLS.guides.modemReinit:
+            hit.cta = "Réinitialiser mon modem";
+            hit.name = "Reboot_Modem";
+            break;
+        default: break;
+        }
+
+        console.log(hit);
```
Comment (antleblanc) [inline client/app/telecom/dashboard/guides/telecom-dashboard-guides.controller.js]: -`console.log`

File: client/app/telecom/sms/order/telecom-sms-order.controller.js [hunk 2]
```diff
@@ -173,6 +174,13 @@ angular.module("managerApp").controller("TelecomSmsOrderCtrl", class TelecomSmsO
             this.Toast.error(this.$translate.instant("sms_order_ko"))
         ).finally(() => {
             this.loading.order = false;
+            return this.atInternet.trackClick({
+                cta: "Générer le bon de commande",
```
Comment (antleblanc) [inline client/app/telecom/sms/order/telecom-sms-order.controller.js]: Are you sure that we have to kept french sentence?

File: client/app/telecom/dashboard/guides/telecom-dashboard-guides.html [hunk 3]
```diff
@@ -8,7 +8,8 @@
                title="{{ ::'telecom_dashboard_FAQ_Activate' | translate }} {{ ::'common_newtab' | translate }}"
                target="_blank"
                rel="noopener"
-               data-ng-href="{{ GuidesCtrl.links.packActivate }}">
+               data-ng-href="{{ GuidesCtrl.links.packActivate }}"
+               data-ng-click="GuidesCtrl.trackRedirection(GuidesCtrl.links.packActivate)">
```
Comment (antleblanc) [inline client/app/telecom/dashboard/guides/telecom-dashboard-guides.html]: I think you will have some side effet with this technic. ref https://github.com/angular/angular.js/issues/6671.

Comment (antleblanc) [inline client/app/telecom/dashboard/guides/telecom-dashboard-guides.html]: I think you will have some side effet with this technic. ref https://github.com/angular/angular.js/issues/6671.

Comment (JeremyDec) [reply client/app/telecom/dashboard/guides/telecom-dashboard-guides.html]: Thx for the ref :)

**Pull request merged**